vehicle_gateway_worlds/package.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>vehicle_gateway_worlds</name>
+  <version>0.0.0</version>
+  <description>Some demo worlds for the Vehicle Gateway</description>
+  <maintainer email="morgan@openrobotics.org">Morgan Quigley</maintainer>
+  <license>Apache License 2.0</license>
+
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
```

Comment:
you added these dependencies but in the `CMakeLists.txt` you not using them. Remove here or add the common linter in the `CMakeLists.txt`